ui-admin/src/populate/PopulateControls.tsx

```diff
@@ -24,32 +24,25 @@ export const OverwriteControl = ({ text, isOverwrite, setIsOverwrite }:
 /**
  * renders a control for specifying a file name --this could be updated to handle the conventions of the various paths
  */
-export const useFileNameControl = () => {
-  const [fileName, setFileName] = useState('')
-  return {
-    fileName,
-    fileNameControl: <label className="form-label">
+export const FileNameControl = ({ fileName, setFileName }: {fileName: string, setFileName: (n: string) => void}) => {
```

Comment:
Nice. I think these make much more sense as components than hooks.

Reply:
yeah, I was palying around with the hooks to see if it could make things more self-contained.  answer: no. :)